modules/core/include/opencv2/core/cvdef.h

```diff
@@ -962,6 +962,9 @@ CV_INLINE int CV_FOURCC(char c1, char c2, char c3, char c4)
     return (c1 & 255) + ((c2 & 255) << 8) + ((c3 & 255) << 16) + ((c4 & 255) << 24);
 }
 
+//! Macro to construct the fourcc code of the codec. Same as CV_FOURCC()
+#define CV_FOURCC_MACRO(c1, c2, c3, c4) (((c1) & 255) + (((c2) & 255) << 8) + (((c3) & 255) << 16) + (((c4) & 255) << 24))
```

Comment:
May be move it to videoio.hpp as inline function (which could be exported to bindings)

BTW, there is VideoWriter::fourcc (it is confusing because it is used outside of that). So global function is preferable.

Reply:
There is `CV_FOURCC` inline function above (used in ~70 places). Do you mean move it too? It is used in _imgproc_ for font rendering params.

Perhaps we can remove `CV_FOURCC_MACRO` completely, I didn't do it to reduce total changes, because it is used in ~50 places.

`VideoWriter::fourcc` is used in ~50 places too.

Reply:
That is a mess.
Lets revise fourcc through a separate task then.